core/arch/arm/kernel/secure_partition.c

```diff
@@ -1351,8 +1353,23 @@ static TEE_Result sp_enter_invoke_cmd(struct ts_session *s,
 	sp_regs->cpsr = read_daif() & (SPSR_64_DAIF_MASK << SPSR_64_DAIF_SHIFT);
 
 	exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
+
+	/*
+	 * Store endpoint ID and thread ID in rpc_target_info. This will be used
+	 * as w1 in FFA_INTERRUPT in case of a NWd interrupt.
```

Comment:
s/NWd inteerupt/foreign interrupt/  (unless NWd is FF-A defined acronym)

Reply:
Done